File: database/src/main/java/bao/webapp/service/OrderService.java
package bao.webapp.service;

import bao.webapp.model.Order;
import java.util.List;

public interface OrderService {

	public void create(Order order);

	public void delete(Order order);

	public Order findById(long id);

	public List<Order> findByUserID(long userID);

	public Iterable<Order> all();
}

File: database/src/main/kotlin/com/ossp/database/service/orders/OrderService.kt
package com.ossp.database.service.orders

import com.ossp.database.model.Order

interface OrderService {

    fun create(order: Order): Order?

    fun delete(order: Order?)

    fun findById(id: Long?): Order?

    fun findByUserID(userID: Long?): List<Order>?

    fun all(): Iterable<Order>?
}